module/metrics/execution.go

```diff
@@ -739,6 +749,11 @@ func (ec *ExecutionCollector) ExecutionTransactionExecuted(
 	ec.transactionExecutionTime.Observe(float64(dur.Milliseconds()))
 	ec.transactionConflictRetries.Observe(float64(numConflictRetries))
 	ec.transactionComputationUsed.Observe(float64(compUsed))
+	if compUsed > 0 {
+		// normalize so the value should be around 1
+		ec.transactionNormalizedTimePerComputation.Observe(
+			(float64(dur.Milliseconds()) / float64(compUsed)) * flow.EstimatedComputationPerMillisecond)
```

Comment:
Precision probably isn't a huge deal, but you may want to do the multiplication first so rounding is more natural.
```suggestion
			float64(dur.Milliseconds() * flow.EstimatedComputationPerMillisecond / float64(compUsed))
```